### PR TITLE
Adding support for namespace in Elastic Search index name

### PR DESCRIPTION
**Pull request by rralcala**
This change adds an option to append the namespace to the index name so that indexes are split by namespace.
This allows to handle larger scale, easier acls and in some cases easier queries.

### CODE REVIEW

File: deploy/Dockerfile [hunk 1]
```diff
@@ -5,9 +5,9 @@ WORKDIR /src/github.com/AliyunContainerService/kube-eventer
 RUN apt-get update -y && apt-get install gcc ca-certificates
 RUN make
 
-FROM alpine:3.10
+FROM golang:1.12.6
```
Comment (ringtail) [inline deploy/Dockerfile]: Why change alpine:3.10 to golang:1.12.6

**Comment by ringtail** [comment]
Thanks for contribution. I think you are very familiar with ES. `kube-eventer` seems to be incompatible with ES6. so could you help on this issue #34 . ☺

File: deploy/Dockerfile (hunk 1 above)
Comment (rralcala) [reply deploy/Dockerfile]: Sorry, that was an accident, I'll remove it.

Comment (ringtail) [reply deploy/Dockerfile]: Thanks

**Comment by fei922** [comment]
Can the index support configuration by month or by day? Now the index only supports by day "$ {baseIndex}-$ {namespace} -yyyy.MM.dd", can it support by month "$ {baseIndex}-$ {namespace } -yyyy.MM "

**Comment by zeusro** [comment]
> Can the index support configuration by month or by day? Now the index only supports by day "$ {baseIndex}-$ {namespace} -yyyy.MM.dd", can it support by month "$ {baseIndex}-$ {namespace } -yyyy.MM "

Maybe you can use elasticsearch pipeline to do this.

See http://www.zeusro.com/2019/04/10/elasticsearch-api/#ingestpipeline-%E7%94%A8%E6%B3%95

**Pull request merged**